replicatesAtOnce.jmd

```diff
@@ -27,7 +42,7 @@ G2_1 = zeros(189, 8, 5)
 G2_2 = zeros(189, 8, 5)
 G2_3 = zeros(189, 8, 5)
 for i=1:5
-    G1_1[:, :, i], G2_1[:, :, i], G1_2[:, :, i], G2_2[:, :, i], G1_3[:, :, i], G2_3[:, :, i] = DrugResponseModel.predict_replicates(ef1[:, :, i], ef2[:, :, i], ef3[:, :, i], concs[:,1], g0)
```

Comment:
Why this change?

Reply:
I had two functions to plot the three replicates along with the simulations; one for the time we were simulating them individually (`plot2()`), and one for the one simulating them at once (`plot_ribbon_replicates()`--I guess). I just deleted one of them to avoid repeated code.